addons/website/static/src/snippets/s_process_steps/options.js

```diff
@@ -0,0 +1,56 @@
+odoo.define('website.s_process_steps_options', function (require) {
+    'use strict';
+    const options = require('web_editor.snippets.options');
+
+    options.registry.ConnectorChoice = options.Class.extend({
```

Comment:
Do not indent the first level of Odoo modules, but in fact you should use the new module syntax. Check other module with the new syntax: `/** @odoo-module`